Solutions/Vectra/Hunting Queries/VectraSuspiciousLongDNSQueries.yaml

```diff
@@ -0,0 +1,31 @@
+id: 3826de45-7300-44d7-800d-a2b457439dda
+name: Vectra AI - Suspicious Long DNS Queries
+description: |
+  'Query searches long DNS queries.
+  A high volume of long DNS queries may indicate the usage of DNS Tunneling.
+  Metadata required = metadata_dns'
+severity: High
+requiredDataConnectors:
+  - connectorId: AIVectraStream
+    dataTypes:
+      - VectraStream
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1071.004
+query: |
+  VectraStream
+  | where metadata_type == "metadata_dns"
+  | extend querylength = strlen(['query'])
+  | summarize count() by querylength, orig_hostname 
+  | sort by querylength desc 
+  | extend HostCustomEntity = orig_hostname, IPCustomEntity = id_orig_h
```

Comment:
@danymello, please check and fix query.
![image](https://user-images.githubusercontent.com/75478517/127637301-42490d69-08d6-4677-9ec6-b314bec793df.png)